kolibri/core/content/utils/import_export_content.py

```diff
@@ -195,7 +195,7 @@ def get_import_export_nodes(  # noqa: C901
         # Only bother with this query if there were any resources returned above.
 
         if nodes_query.count() > 0:
-            nodes_queries_list.append(nodes_query)
+            nodes_queries_list.append(nodes_query.distinct())
```

Comment:
I think this change is totally logical, however, this is breaking the tests.
As `nodes_queries` has an outer join with `File`, it caused repeated appearances of the same node when the node had more than one file in the relation.
Tests need to be changed to reflect the right amount of nodes.

Reply:
Ah thanks - I checked the import export tests, but not the device API tests.